core-java/src/main/java/org/spine3/server/aggregate/Aggregate.java

```diff
@@ -220,22 +225,31 @@ public void play(Iterable<EventRecord> records) throws InvocationTargetException
         }
     }
 
+    /**
+     * Applies events to an aggregate unless they are state-neutral.
+     *
+     * <p>Event applier should call {@link #incrementState(Message)},
```

Comment:
We need to have description of an event applier somewhere else. Since this method is private it's not really likely that this documentation is going to be read soon enough. But the user would need to understand how to write an applier from the very beginning.